scripts/zones/Riverne-Site_B01/mobs/Bahamut.lua

```diff
@@ -26,7 +26,8 @@ entity.onMobSpawn = function(mob)
     -- base damage of 136, so (lvl 83 + 2) * 1.60
     mob:setMobMod(xi.mobMod.WEAPON_BONUS, 160)
     -- gives firaga iv a cast time of ~2 seconds as per retail
-    mob:setMod(xi.mod.UFASTCAST, 75)
+    -- note baha has a job trait with fast cast of 15% so 75% total
+    mob:setMod(xi.mod.UFASTCAST, 60)
```

Comment:
where is the other 15% coming from?  is bahamut part rdm?

Reply:
> where is the other 15% coming from? is bahamut part rdm?

Bahamut is BLM/RDM in the code